### PR TITLE
Bug 1794096: CRD storageversionmigrations.migration.k8s.io has no schema

### PR DESCRIPTION
**Pull request by sanchezl**

### CODE REVIEW

**Comment by sttts** [comment]
Is there a corresponding a upstream PR?

File: manifests/0000_40_kube-storage-version-migrator-operator_01_crd.yaml [hunk 1]
```diff
@@ -1,43 +0,0 @@
...
-  - name: v1alpha1
-    served: true
-    storage: true
----
-apiVersion: apiextensions.k8s.io/v1beta1
-kind: CustomResourceDefinition
-metadata:
-  name: storagestates.migration.k8s.io
-  annotations:
-    "api-approved.kubernetes.io": "https://github.com/kubernetes/kubernetes/pull/0000"
```
Comment (sttts) [inline manifests/0000_40_kube-storage-version-migrator-operator_01_crd.yaml]: please don't mix moves and changes. Make 2 commits for that.

This annotation is gone but shouldn't?

Comment (sanchezl) [reply manifests/0000_40_kube-storage-version-migrator-operator_01_crd.yaml]: Changed the value to `unsupported` which is more correct.

Comment (sanchezl) [reply manifests/0000_40_kube-storage-version-migrator-operator_01_crd.yaml]: Split into two commits.

Comment (sttts) [reply manifests/0000_40_kube-storage-version-migrator-operator_01_crd.yaml]: Sorry for being picky. The change might be fine, but deserves another commit. It's not about splitting or moving, but a real change. Now it looks like it accidently is part of the commit.

Comment (sanchezl) [reply manifests/0000_40_kube-storage-version-migrator-operator_01_crd.yaml]: Done

**Comment by sttts** [comment]
The upstream PR has a Makefile change to download the generator. Will be pick that change here too? Or do we completely switch back to upstream?

**Comment by sanchezl** [comment]
> The upstream PR has a Makefile change to download the generator. Will be pick that change here too? Or do we completely switch back to upstream?

I will pick it up in a future merge.

**Comment by sanchezl** [comment]
> Is there a corresponding a upstream PR?

https://github.com/kubernetes-sigs/kube-storage-version-migrator/pull/56

**Pull request merged**